### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jaapjansma**
Fixed typo in readme

### CODE REVIEW

**Comment by tafaRU** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [![Build Status](https://travis-ci.org/OCA/partner-contact.svg?branch=8.0)](https://travis-ci.org/OCA/partner-contact)
 [![Coverage Status](https://coveralls.io/repos/OCA/partner-contact/badge.png?branch=8.0)](https://coveralls.io/r/OCA/partner-contact?branch=8.0)
 
-OCA parter and contact management modules for Odoo
+OCA partner and contact management modules for Odoo
 ==================================================
```
Comment (pedrobaeza) [inline README.md]: You must expand this line also one character.

**Comment by jaapjansma** [comment]
I have updated the line below also

**Comment by pedrobaeza** [comment]
Thanks. I merge due to the triviality.

**Comment by yvaucher** [comment]
Could have been done in one commit :-/

**Comment by pedrobaeza** [comment]
You're right. I have fixed it including the change in previous commit and forcing a push.

**Comment by yvaucher** [comment]
@pedrobaeza thanks

**Pull request merged**